app/javascript/components/actionable.js

```diff
@@ -48,6 +48,18 @@ export default template({
         value: this.statement.verified_on, type: 'time'
       }
 
+      if (this.page.reference_url_title) {
+        reference[wikidataClient.getPropertyID('title')] = {
```

Comment:
There's a typo here: `reference` should be `references` (and it's the same inside the next `if`)